cmd/manager/main.go

```diff
@@ -87,7 +93,7 @@ func run() error {
 	// Create a new Cmd to provide shared dependencies and start components
 	mgr, err := manager.New(cfg, manager.Options{
 		Namespace:          "", //watch all namespaces
-		MapperProvider:     restmapper.NewDynamicRESTMapper,
+		MapperProvider:     func(cfg *rest.Config) (meta.RESTMapper, error) { return apiutil.NewDynamicRESTMapper(cfg) },
```

Comment:
Note to reviewers: The [docs](https://sdk.operatorframework.io/docs/upgrading-sdk-version/version-upgrade-guide/#v014x) said s/restmapper/apiutil/ was a drop-in replacement. They lied.